src/main/java/au/gov/geoscience/portal/server/services/filters/PetroleumTenementFilter.java

```diff
@@ -33,16 +33,15 @@ public PetroleumTenementFilter(String name, String holder) {
      * @param tenementTypeUris - type of tenement
      */
     public PetroleumTenementFilter(String name, String holder, Set<String> statusUris, Set<String> tenementTypeUris) {
-        PetroleumTenementServiceProviderType petroleumTenementServiceProviderType = PetroleumTenementServiceProviderType.GeoServer;
-        fragments = new ArrayList<String>();
+        fragments = new ArrayList<>();
         if (name != null && !name.isEmpty()) {
-            fragments.add(this.generatePropertyIsLikeFragment(petroleumTenementServiceProviderType.nameField(), name));
```

Comment:
Does this change speak to a problem with PetroleumTenementServiceProviderType that should be fixed, instead of just not using it?

Reply:
Although I see it's already not used in other places in this file, so I guess that's non-issue.

Reply:
PetroleumTenementServiceProviderType was based on the mineral-tenements equivalent which was used for differentiating between ArcGIS and Geoserver.

Petroleum only uses Geoserver so I'll rework this serviceProviderType or remove it all together